src/test/java/systemtests/AddressBookSystemTest.java

```diff
@@ -276,7 +281,9 @@ private void assertApplicationStartingStateIsCorrect() {
             assertEquals("", getCommandBox().getInput());
             assertEquals("", getResultDisplay().getText());
             assertListMatching(getPersonListPanel(), getModel().getFilteredPersonList());
-            assertEquals(MainApp.class.getResource(FXML_FILE_FOLDER + DEFAULT_PAGE), getBrowserPanel().getLoadedUrl());
+            assertTaskListMatching(getTaskListPanel(), getModel().getFilteredTaskList());
+            //assertEquals(MainApp.class.getResource(FXML_FILE_FOLDER + DEFAULT_PAGE),
```

Comment:
Try not to comment out test-cases to make it "work" as far as possible, as you run the risk of accidentally missing them out in the future, and you lose test-coverage as well.